doc/mobile/cross_compiling_for_android_cn.md

```diff
@@ -20,10 +20,32 @@ $ docker build -t username/paddle-android:dev . -f Dockerfile.android
 构建好开发镜像后，即可使用开发镜像来编译Android版PaddlePaddle C-API库。
 Android的Docker开发镜像向用户提供两个可配置的参数：
 
-| Argument        | Optional Values         | Default |
-|-----------------|-------------------------|---------|
-|`ANDROID_ABI`    |`armeabi-v7a, arm64-v8a` | `armeabi-v7a` |
-|`ANDROID_API`    |`>= 21` | `21` |
+<table class="docutils">
```

Comment:
Why do we change above Markdown table format into HTML format?

Reply:
@jetfuel If we use Markdown table format, can we display it on our website?

Reply:
@wangkuiyi Because the Markdown table is ugly on the old website `doc.paddlepaddle.org`.